### PR TITLE
added authors list as per ISIS_TC action item,

### PR DESCRIPTION
**Pull request by AndrewAnnex**
## Description
Adds Authors.rst file to the project, includes initial list of contributors to isis3 that are present in the git history, with some minor edits to remove apparent duplicates of usernames and bots.

I need to figure out some first name only authors still, maintainers are free to edit as needed to add missing people. I am. not sure if the git history as recorded here is the full contribution history of ISIS3, so if there is an older SVN to investigate let me know.

bonus: I really wanted the AUTHORS.rst to be a quine, but the first two things I tried didn't work so it is likely not worth it.

## Related Issue
none that I am aware of

## Motivation and Context
this was motivated by an discussion from the April ISIS_TC meeting

## How Has This Been Tested?
NA

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [x] Documentation change (update to the documentation; no code change)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] I have read and agree to abide by the [Code of Conduct](https://usgs-astrogeology.github.io/code/)
- [x] I have read the [**CONTRIBUTING**](https://github.com/USGS-Astrogeology/ISIS3/blob/dev/CONTRIBUTING.md) document.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.

## Licensing
This project is mostly composed of free and unencumbered software released into the public domain, and we are unlikely to accept contributions that are not also released into the public domain. Somewhere near the top of each file should have these words:

> This work is free and unencumbered software released into the public domain. In jurisdictions that recognize copyright laws, the author or authors of this software dedicate any and all copyright interest in the software to the public domain.

- [x] I dedicate any and all copyright interest in this software to the public domain. I make this dedication for the benefit of the public at large and to the detriment of my heirs and successors. I intend this dedication to be an overt act of relinquishment in perpetuity of all present and future rights to this software under copyright law.

### CODE REVIEW

**Review by rbeyer — DISMISSED** [review]
I appreciate the explanation at the bottom of how you generated this, but it is irrelevant as soon as the first manual edit happens, so I'd get rid of it, just so people don't think they are limited to using it or something.

**Review by rbeyer — DISMISSED** [review]

File: AUTHORS.rst [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+- Kris Becker 
+- Kristin Berry 
+- Lauren Adoram-Kershner 
+- Lee 
+- Lynn Weller 
+- Mackenzie Boyd 
+- Makayla Shepherd 
+- Marjorie Hahn 
+- Mathew Eis 
+- Michael Aye
```
Comment (jessemapel) [inline AUTHORS.rst:46]: @michaelaye You're on here as both Michael Aye and K. Michael Aye. Do you have a preference?

File: AUTHORS.rst [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+- Kaitlyn Lee 
+- Kaj E Williams  
+- Kelvin Rodriguez 
+- Ken Edmundson 
+- Kim Oyama  
+- K.-Michael Aye 
+- Kris Becker 
+- Kristin Berry 
+- Lauren Adoram-Kershner 
+- Lee
```
Comment (jessemapel) [inline AUTHORS.rst]: I think this is Kaitlyn Lee, but I'm not sure.

**Comment by AndrewAnnex** [comment]
I think we just need to work out who `oxez` is

File: AUTHORS.rst (hunk 1 above)
Comment (michaelaye) [reply AUTHORS.rst:46]: Yeah, i recently changed my GH persona to just be Michael Aye, as I don't use the first part of my name, so Michael Aye would be fine, but I don't mind much.

**Comment by jessemapel** [comment]
oxez is a duplicate of Cole Neubauer:


```
git log --pretty="- %ae %an" | sort | uniq -i
...
- cneubauer@live.com Cole Neubauer
- cneubauer@live.com Oxez
- cneubauer@usgs.gov Cole Neubauer
```

**Comment by jessemapel** [comment]
I think keeping how this was made originally is nice, but it could be documented on the PR instead of in the file itself. I leave the decision of what to do up to Andrew.

**Review by jessemapel — APPROVED** [review]

**Comment by jessemapel** [comment]
Also, this has all of the old SVN commits in it because we used git-svn to generate the original git repo. There are some commits that aren't here because of how the branches in SVN were handled, but it's rather complicated to automate pulling that information and I don't think it would catch any new authors.

**Review by michaelaye — APPROVED** [review]

**Comment by AndrewAnnex** [comment]
@jessemapel good to know that everyone is included, I personally like having instructions in the file for posterity's sake. Although I don't feel strongly about keeping it, I feel more strongly about keeping it in rather than removing it if that makes sense.

**Review by krlberry — APPROVED** [review]

**Review by rbeyer — APPROVED** [review]

**Pull request merged**